pages/windows/color.md

```diff
@@ -14,3 +14,17 @@
 - Set the console foreground and background to a specific color:
```

Comment:
```suggestion
- Set the console foreground and background to a specific color (allowed values: `1-9, a-f`):
```

Reply:
Maybe instead mention it should be a Hexadecimal number

Reply:
```suggestion
- Set the console foreground and background to a specific color using hexadecimal numbers (`1-9,a-f`):
```